packages/amplication-data-service-generator/src/server/seed/seed.template.ts

```diff
@@ -41,7 +41,7 @@ async function seed(bcryptSalt: Salt) {
     update: {},
     create: data,
   });
-  client.$disconnect();
+  void client.$disconnect();
```

Comment:
why did you add this?

Reply:
the lint screamed at me and asked me to add this 😅
look on [17004e5](https://github.com/amplication/amplication/pull/2295/commits/17004e528fad61d6124bdb5ce4c72facedeea14c)